docs/python_docs/python/Makefile_sphinx

```diff
@@ -18,8 +18,25 @@
 # Makefile for Sphinx documentation
 #
 
+
+NPROCS := 1
+OS := $(shell uname)
+export NPROCS
+
+ifeq ($J,)
+
+ifeq ($(OS),Linux)
+  NPROCS := $(shell grep -c ^processor /proc/cpuinfo)
```

Comment:
can we use nproc here?

Reply:
I tried that first, but I guess you can't do that in a Makefile... so I looked it up and found this referenced a couple of times. It did the trick...

Reply:
ok, makes sense.